pkg/blockchain/interface.go

```diff
@@ -47,3 +43,8 @@ type NodeRegistry interface {
 		httpAddress string,
 	) error
 }
+
+type IMessagePublisher interface {
+	PublishIdentityUpdate(ctx context.Context, inboxId [32]byte, identityUpdate []byte) error
+	PublishGroupMessage(ctx context.Context, groupdId [32]byte, message []byte) error
```

Comment:
PublishCommit?

Reply:
Technically it doesn't _have_ to be a commit. If you really really want censorship resistance you could just publish everything here